stdlib/public/core/StringProtocol.swift

```diff
@@ -29,6 +29,8 @@ public protocol StringProtocol
 
   associatedtype UnicodeScalarView : BidirectionalCollection
   where UnicodeScalarView.Element == Unicode.Scalar
+  
+  associatedtype SubSequence = Substring
```

Comment:
I think you want an `override` keyword on that `associatedtype`? Didn't you get a warning about that?

Reply:
Nope!

Reply:
Do you need an `override` if it's doing something new like setting a new default? `Collection.SubSequence = Slice` doesn't have one either.